html/modules/custom/ghi_blocks/src/Plugin/Block/Plan/PlanOperationalPresenceMap.php

```diff
@@ -104,11 +101,12 @@ public function buildContent() {
     $chart_id = Html::getUniqueId('plan-operational-presence-map');
     $chart_class = Html::getClass('plan-operational-presence-map');
     $selected_view = $this->getSelectedView();
+    $langcode = $this->getCurrentPlanObject()->getPlanLanguage() ?? 'en';
 
     $map_settings = [
       'json' => $map_data,
       'id' => $chart_id,
-      'disclaimer' => $conf['display']['disclaimer'] ?: self::DEFAULT_DISCLAIMER,
+      'disclaimer' => $conf['display']['disclaimer'] ?: $this->getPlanMapDisclaimer($langcode),
```

Comment:
Just thinking about this function name. Better change it to `getDefaultMapDisclaimer()` everywhere.